File: 01-50/01/1.fs
VARIABLE SUM
: run
    0 SUM !
    1000 1 DO
        I 5 MOD
        0 = IF
            I SUM @ + SUM !
        ELSE
            I 3 MOD
            0 = IF I SUM @ + SUM ! THEN
        THEN
    LOOP
    SUM @ . CR ;
run
bye

File: 01-50/01/1.cs
using System;
 
public class Euler {
    static public void Main() {
        int sum = 0;
        for (int i = 0; i < 1000; i++) {
            if (i % 5 == 0 || i % 3 == 0) {
                sum += i;
            }
        }
        Console.Out.WriteLine(sum);
    }
}
